packages/frameworks/package.json

```diff
@@ -2,5 +2,13 @@
   "name": "@vercel/frameworks",
   "version": "0.0.14",
   "main": "frameworks.json",
-  "license": "UNLICENSED"
+  "license": "UNLICENSED",
+  "scripts": {
+    "test-unit": "jest --env node --verbose --runInBand --bail test/*unit.*test.*"
+  },
+  "devDependencies": {
+    "@types/jest": "25.2.2",
+    "@types/node": "14.0.1",
+    "jest": "26.0.1"
```

Comment:
Let's use the same version as now-build-utils to avoid running multiple versions of jest.

```
"@types/node": "12.0.4",
"jest": "24.9.0",
"ts-jest": "24.1.0",
"typescript": "3.5.2",
```

This should prevent any update to `yarn.lock`